doc/api/crypto.markdown

```diff
@@ -768,6 +768,25 @@ default, set the `crypto.DEFAULT_ENCODING` field to 'binary'.  Note
 that new programs will probably expect buffers, so only use this as a
 temporary measure.
 
+## Caveats
+
+The crypto module still supports some algorithms which are already
+compromised. And the API also allows to use ciphers and hashes with a
+small key size that are considered to be too weak for safe use.
+
+Users should take full responsibility for selecting the crypto
+algorithm and key size according to their security requirements.
+
+The following descriptions only show some recommendations to use
+crypto module. See [NIST SP 800-131A] for details.
+
```

Comment:
Perhaps something like, "Based on the recommendations of [NIST SP 800-131A]:" instead?